src/web/components/Box/box.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Container from './';
```

Comment:
Box or Container ?

Reply:
It's a box. sorry for the copy/paste mistake